testing-commandments.md

```diff
@@ -0,0 +1,22 @@
+# HolidayExtras Automated Testing Commandments
+
+## Thou shalt be consistent within projects
+If a project has a de-facto standard, it should be maintained. The process for changing a de-facto standard should take the form of a whiteboard meeting, not a pull request.
+
+## Thy tests should be obvious
+If the test behaviour isn't obvious by looking at it, it's no good. If I have to scroll for days to read a test, it's no good.
+
+## Thy tests should be loosely coupled with thy code
```

Comment:
What if, for example, I change the return value of a time orientation module from a timestamped integer to a `Date` object? That is a code change and if that return value is under test, which it should be, my tests will require changing. Which is ok.

I assume by code, you mean code implementation, when the interface could also be interpreted as "code".